### DISCUSSION TITLE
I can't compile pkgi-ps3

### DISCUSSION THREAD

**Discussion by Leonidas-source** [bucanero/pkgi-ps3#96] [Q&A]
I created script to compile pkgi(https://github.com/Leonidas-source/PS3-pkgi-builder) but I always get this error
"pkgi_ps3.c
/home/user/build/pkgi-ps3/source/pkgi_ps3.c: In function 'init_music':
/home/user/build/pkgi-ps3/source/pkgi_ps3.c:231:28: error: 'drv_psl1ght' undeclared (first use in this function); did you mean 'drv_sgi'?
     MikMod_RegisterDriver(&drv_psl1ght);
                                              ^~~~~~~~~~~
                                              drv_sgi
/home/user/build/pkgi-ps3/source/pkgi_ps3.c:231:28: note: each undeclared identifier is reported only once for each function it appears in
make[1]: *** [/usr/local/ps3dev//base_rules:69: pkgi_ps3.o] Error 1
make: *** [Makefile:129: build] Error 2"
How I could fix this issue?

**Comment by bucanero** [comment]
It looks like there's an issue with the `mikmod` library, or it's not properly installed. The `drv_psl1ght` driver should be part of the library if it's installed as part of the ps3 toolchain.

**Reply by Leonidas-source** [reply]
I'm not sure where is a problem in my script(https://github.com/Leonidas-source/PS3-pkgi-builder/blob/main/install.sh). Could you please check lines 24-30, 61-68?
For my opinion it's all good. Maybe I forgot something?

**Reply by bucanero** [reply]
I can't say, there's a github action script that builds pkgi and has no issues: https://github.com/bucanero/pkgi-ps3/blob/master/.github/workflows/build.yml
